### PR TITLE
🐛 [bitnami/airflow] - fixing node selector, toleration and affinity

### PR DESCRIPTION
**Pull request by bappr**
**Description of the change**

Fix an issue when node selector, toleration or affinity is enabled

**Benefits**

Fix.

**Possible drawbacks**

None.

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [ ] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [ ] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/airflow/templates/deployment-scheduler.yaml [hunk 1]
```diff
@@ -26,24 +26,24 @@ spec:
...
-    {{- end }}
     spec:
       {{- include "airflow.imagePullSecrets" . | nindent 6 }}
       {{- if .Values.securityContext.enabled }}
       securityContext:
         fsGroup: {{ .Values.securityContext.fsGroup }}
         runAsUser: {{ .Values.securityContext.runAsUser }}
       {{- end }}
+      {{- if .Values.nodeSelector }}
+      nodeSelector: {{- toYaml .Values.nodeSelector | nindent 8 }}
```
Comment (sameersbn) [inline bitnami/airflow/templates/deployment-scheduler.yaml]:
```suggestion
      nodeSelector: {{- toYaml .Values.nodeSelector | nindent 10 }}
```

File: bitnami/airflow/templates/deployment-scheduler.yaml [hunk 2]
```diff
@@ -26,24 +26,24 @@ spec:
...
+      {{- end }}
+      {{- with .Values.affinity }}
+      affinity:
+      {{ toYaml . | nindent 8 }}
```
Comment (sameersbn) [inline bitnami/airflow/templates/deployment-scheduler.yaml]:
```suggestion
      {{ toYaml . | nindent 10 }}
```

File: bitnami/airflow/templates/deployment-scheduler.yaml [hunk 3]
```diff
@@ -26,24 +26,24 @@ spec:
...
+      {{- end }}
+      {{- with .Values.tolerations }}
+      tolerations:
+      {{ toYaml . | nindent 8 }}
```
Comment (sameersbn) [inline bitnami/airflow/templates/deployment-scheduler.yaml]:
```suggestion
      {{ toYaml . | nindent 10 }}
```

File: bitnami/airflow/templates/deployment-web.yaml [hunk 4]
```diff
@@ -26,24 +26,24 @@ spec:
...
-    {{- end }}
     spec:
       {{- include "airflow.imagePullSecrets" . | nindent 6 }}
       {{- if .Values.securityContext.enabled }}
       securityContext:
         fsGroup: {{ .Values.securityContext.fsGroup }}
         runAsUser: {{ .Values.securityContext.runAsUser }}
       {{- end }}
+      {{- if .Values.nodeSelector }}
+      nodeSelector: {{- toYaml .Values.nodeSelector | nindent 8 }}
```
Comment (sameersbn) [inline bitnami/airflow/templates/deployment-web.yaml]:
```suggestion
      nodeSelector: {{- toYaml .Values.nodeSelector | nindent 10 }}
```

File: bitnami/airflow/templates/deployment-web.yaml [hunk 5]
```diff
@@ -26,24 +26,24 @@ spec:
...
+      {{- end }}
+      {{- with .Values.affinity }}
+      affinity:
+      {{ toYaml . | nindent 8 }}
```
Comment (sameersbn) [inline bitnami/airflow/templates/deployment-web.yaml]:
```suggestion
      {{ toYaml . | nindent 10 }}
```

File: bitnami/airflow/templates/deployment-web.yaml [hunk 6]
```diff
@@ -26,24 +26,24 @@ spec:
...
+      {{- end }}
+      {{- with .Values.tolerations }}
+      tolerations:
+      {{ toYaml . | nindent 8 }}
```
Comment (sameersbn) [inline bitnami/airflow/templates/deployment-web.yaml]:
```suggestion
      {{ toYaml . | nindent 10 }}
```

File: bitnami/airflow/Chart.yaml [hunk 7]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: airflow
-version: 4.4.2
+version: 4.4.3
```
Comment (sameersbn) [inline bitnami/airflow/Chart.yaml]:
```suggestion
version: 4.5.0
```

**Comment by bappr** [comment]
Hey @sameersbn thanks for reviewing it.

Why changing the indent?

Also, this is a patch, not a minor upgrade.
Why can't I use `4.4.3`?

**Comment by sameersbn** [comment]
> Why changing the indent?

ah you're right. the block were just put in the wrong place it appears



> Also, this is a patch, not a minor upgrade.

fair enough

**Pull request merged**